recipe/meta.yaml

```diff
@@ -10,34 +10,48 @@ source:
   sha256: fe859b0e98d222b4b458022007daac2757e970f0671d45e55d01931c1b4bb3cb
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  noarch: python
```

Comment:
```suggestion
```

This package contains compiled code. It shouldn't be delivered as noarch.
(noarch would mean the linux-64 build getting used on all architectures.